pyccl/__init__.py

```diff
@@ -16,6 +16,12 @@
 # SWIG-generated
 from . import ccllib as lib
 
+# monkey patch for isitgr and fast-pt if Numpy>=1.24
+from packaging.version import parse
+import numpy as np
+if bool(parse(np.__version__) >= parse('1.24')):
```

Comment:
One final thing I just realized. Comparison with the output of the `parse` function will return a boolean, so there's no need to convert it to one.